### ISSUE TITLE
Export Lossless Video and Audio. FLAC crashes OpenShot

### ISSUE THREAD

**Issue by OpenShotUser80** [OpenShot/openshot-qt#2252]
Hello there.

I recently downloaded OpenShot Video Editor and think it's very good software for video editing. You can do a lot with it and it's very easy to understand how to use. Stunning videos can be created easily.
I think the software is nearly perfect apart from being unable to export lossless. I wish that it could export to AVI using the HuffYUV, MagicYUV or the UT Video codecs. Also the same for audio. It crashes if I try to export to FLAC, but I'd prefer WAV and I don't think OpenShot can export to WAV yet as it says "cannot write header to file".
Please would you consider adding lossless export features?

**Comment by DylanC** [comment]
Dup of #1480

**Comment by DylanC** [comment]
@OpenShotUser80 - What version of Openshot did you use?

**Comment by peanutbutterandcrackers** [comment]
Lossless export had been worked upon, but the patch has gotten out of date. Hopefully, we'll have it in the main source soon. :+1:

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Still a problem in v2.4.3. Do you remember where we had the discussion on this issue?

**Comment by OpenShotUser80** [comment]
Thanks for the replies.
I'm using v2.4.3 and it's works great as long as the settings are correct in the export window or else it closes itself.
Glad to hear that lossless export had been worked on. Like you say hopefully it will be back in OpenShot in the near future. I would actually pay for this software it's that good.

**Comment by peanutbutterandcrackers** [comment]
@DylanC - It wasn't really discussed. I got the patch from peterm just to test it out and I did. It seemed to work just great. Used x264's crf 0 to do the lossless export in .mp4 container. Perhaps it'll be there again with the hw_accel branch's merge? But [here is a work-around to achieve lossless export](https://github.com/OpenShot/openshot-qt/issues/750#issuecomment-387874047) in OpenShot.

Hope that helps!
